src/Scan/Scans.ts

```diff
@@ -159,6 +159,28 @@ export interface ScanState {
   issuesBySeverity: CountIssuesBySeverity[];
 }
 
+export enum SourceType {
+  OPEN_API = 'openapi',
+  RAML = 'raml',
+  POSTMAN = 'postman',
+  HAR = 'har'
+}
+
+export interface StorageFile {
+  storageId: string;
+  name: string;
+  size: number;
+  mimeType: string;
+  path: string;
+  type: SourceType;
+  revision?: number;
+  lastModified?: Date;
+  id?: string;
+  url?: string;
+  harPath?: string;
+  readyToBeUsed?: boolean;
```

Comment:
I don't see any reason to define a while fields, especially when you use only one of them. Remove all of them except `id` and `type`